source/cli.js

```diff
@@ -25,6 +25,7 @@ const cli = meow(`
 	  --no-cleanup        Skips cleanup of node_modules
 	  --yolo              Skips cleanup and testing
 	  --no-publish        Skips publishing
+	  --preview           Skips publishing and pushing
```

Comment:
A more accurate description would be something along the lines of:

```suggestion
	  --preview           Only show what np is going to execute without running anything
```